LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2LWAbilitiesModTemplate.uc

```diff
@@ -383,7 +390,26 @@ static function UpdateShadow(X2AbilityTemplate Template)
 
 	Template.AddTargetEffect(ToHitModifier);
 }
+static function UpdateBayonet(X2AbilityTemplate Template)
```

Comment:
Could we also please have a blank line between all top level constructs (functions, defaultproperties, struct definitions, etc.)?